src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-list/cephfs-list.component.html

```diff
@@ -1,29 +1,58 @@
-<cd-table [data]="filesystems"
-          columnMode="flex"
-          [columns]="columns"
-          (fetchData)="loadFilesystems($event)"
-          identifier="id"
-          forceIdentifier="true"
-          selectionType="single"
-          [hasDetails]="true"
-          (setExpandedRow)="setExpandedRow($event)"
-          (updateSelection)="updateSelection($event)">
-  <cd-cephfs-tabs *cdTableDetail
-                  [selection]="expandedRow">
-  </cd-cephfs-tabs>
-  <div class="table-actions">
-    <cd-table-actions [permission]="permissions.cephfs"
-                      [selection]="selection"
-                      class="btn-group"
-                      id="cephfs-actions"
-                      [tableActions]="tableActions">
-    </cd-table-actions>
-  </div>
-</cd-table>
+<cds-tabs
+  [type]="'contained'"
+  [followFocus]="true"
+  [isNavigation]="false"
+  [cacheActive]="true"
+  >
+  <cds-tab
+    heading="File Systems"
```

Comment:
I believe texts should be sentence-case capitalized: https://carbondesignsystem.com/guidelines/content/writing-style/#use-sentence-case-capitalization

And also, to add internationalization and have translatable texts, we should explicitly add it to the property we want so: `i18n-heading`
```suggestion
    heading="File systems"
    i18n-heading
```

Reply:
I have made the changes accordingly